gateway/assets/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <link href="https://fonts.googleapis.com/css?family=Rationale|Roboto+Mono|Roboto:300,400,400i,500,700" rel="stylesheet">
     <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/angular_material/1.1.0/angular-material.min.css">
-    <link rel="stylesheet" href="style/bootstrap.css">
+    <link rel="stylesheet" href="style/bootstrap.css?@@bootstrap.css@@">
```

Comment:
Haven't checked but does this path work while developing locally? (mounting the `index.html` directly to the gateway)

Reply:
it's preferable to use `build.sh` to have the placeholders replaced with hash, but if not doing so, it will still work

Reply:
nice! 👍